### DISCUSSION TITLE
blog/countdown-v0.4.0/

### DISCUSSION THREAD

**Discussion by giscus** [gadenbuie/garrickadenbuie-com#126] [Comments]
# blog/countdown-v0.4.0/

countdown v0.4.0 is now available on CRAN with a ton of new features!

https://www.garrickadenbuie.com/blog/countdown-v0.4.0/

**Comment by shannonpileggi** [comment]
The {countdown} package is so awesome! Thank you for developing this feature!

**Comment by ddsjoberg** [comment]
I love countdown. Everytime I teach a workshop, I turn to it! Thank you!

**Reply by gadenbuie** [reply]
Thanks Daniel! Hope your workshop at conf goes well!
